src/groupeddataframe/grouping.jl

```diff
@@ -234,9 +320,9 @@ combine(d -> sum(skipmissing(d[:c])), gd)
 of `combine(map(f, groupby(df, cols)))`.
 
 """
-function combine(f::Function, gd::GroupedDataFrame)
+function combine(f::Union{Function, Type, Pair}, gd::GroupedDataFrame)
```

Comment:
A general comment (although I know I have changed it earlier to `Union{Function, Type}`). Maybe we should remove type restriction on `f` entirely in all those methods - this is what Base does. Later we can only check if `f` is a `Pair`. In this way we would allow users to pass functors as `f` (which is currently disallowed).